Source/objects.cpp

```diff
@@ -4028,15 +4026,13 @@ void SyncOpL2Door(int pnum, int cmd, int i)
 
 void SyncOpL3Door(int pnum, int cmd, int i)
 {
-	BOOL do_sync;
+	DIABOOL do_sync;

```

Comment:
lol, do we actually have `DIABOOL` defined now? :p

Reply:
ofc, it's been there for a long time xD